gcloud-java-examples/src/main/java/com/google/cloud/examples/nio/CountBytes.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.examples.nio;
+
+import com.google.common.base.Stopwatch;
+
+import java.io.IOException;
+import java.net.URI;
+import java.nio.ByteBuffer;
+import java.nio.channels.SeekableByteChannel;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * CountBytes will read through the whole file given as input.
+ *
+ * <p>It's meant for testing that NIO doesn't read too slowly.
```

Comment:
I still don't like this. I would prefer to describe what the application does rather than stating that it's meant to evaluate performance (which is very opinable), something like:

> This example shows how to read a file size using NIO. `File.size` returns the size of the file as saved in Storage metadata. This class also shows how to compute the file's size by reading all bytes through a channel.